tools/run_build_bot.py

```diff
@@ -223,19 +223,21 @@ def test_request(merge_request):
 
     error_lines += test_output("nosetests -v", title="UNIT TESTS", test=lambda x: x.endswith("OK\n"))
     error_lines += test_output("flake8 .", title="FLAKE8")
-    error_lines += test_output('git diff HEAD^ HEAD|grep -P "^(?i)\+((.*#\s*NOQA:?\s*|(\s*#\s*flake8:\s*noqa\s*))$"',
-                               title="BROAD NOQA'S")
-    noqas = test_output('git diff HEAD^ HEAD|grep -P "^(?i)\+.*#\s*NOQA:\s*[a-z][0-9]{0,3}(\s*,\s*[a-z][0-9]{0,3})*$"',
-                        title="JUST NOQA'S")
+    noqas = test_output('git diff HEAD^ HEAD|grep -P "^(?i)\+((.*#\s*NOQA:?\s*|(\s*#\s*flake8:\s*noqa\s*))$"',
```

Comment:
I will merge this pull request.
Then you can fix this part. Because 'grep -P' always fails. But the error lines are not appened.

[wguan_local@aipanda144 pilot-2.0]$ sh -c 'git diff HEAD^ HEAD|grep -P "^(?i)\+.*#\s*NOQA:\s*[a-z][0-9]{0,3}(\s*,\s*[a-z][0-9]{0,3})*$"'
grep: -P supports only unibyte and UTF-8 locales